.github/workflows/ci.yml

```diff
@@ -45,6 +45,13 @@ jobs:
     - name: Install isal
       if: matrix.with-isal && !startsWith(matrix.os, 'macos')
       run: sudo apt-get install isal libisal-dev
+    - name: Install pbzip2
```

Comment:
Please do not use the bash statements. but use the yaml `if` clause instead. You may need to create two separate tasks (one for mac, one for linux) but that is okay.

Reply:
I agree, but this is copied from how we did it in the "Install pigz" step just above it. I also suggest to install both `pigz` and `pbzip2` in the same step.